crypto/x509v3/v3_ia5.c

```diff
@@ -90,7 +90,7 @@ char *i2s_ASN1_IA5STRING(X509V3_EXT_METHOD *method, ASN1_IA5STRING *ia5)
 }
 
 ASN1_IA5STRING *s2i_ASN1_IA5STRING(X509V3_EXT_METHOD *method,
-                                   X509V3_CTX *ctx, char *str)
+                                   X509V3_CTX *ctx, const char *str)
```

Comment:
You're right, the cast is useless.
The second argument is typed 'const void* ' ...  and it is directly recast to 'const char *' on the other side.
I removed it.